### ISSUE TITLE
Why the maxInstalls count and google play console stats are not matching ?

### ISSUE THREAD

**Issue by usman10scorpio** [facundoolano/google-play-scraper#415]
I have posted this before but @facundoolano did not answer my question. I have checked this with 2 apps.

I compare the **maxInstalls** count with **google play console stats report** and they dont match at all. 

Here are the steps i followed.

I set first dropdown to user acquisitions 
I set the time intervals to monthly
I set the calendar dropdown to lifetime.

![image](https://user-images.githubusercontent.com/12570317/85413364-2b00ba80-b584-11ea-83e1-8bdec2b1b7af.png)

![image](https://user-images.githubusercontent.com/12570317/85413373-2e944180-b584-11ea-958f-197c451f4e08.png)
 

Then I download the csv report,sum all the stats and here is the result.

![image](https://user-images.githubusercontent.com/12570317/85413563-64392a80-b584-11ea-8152-e57cacd1a0b9.png)

and this is what I have got through **npm package**

![image](https://user-images.githubusercontent.com/12570317/85413690-8f237e80-b584-11ea-8f21-6c7d90ed93d0.png)

It clearly shows they dont match at all.

I take **maxInstalls as total no of downloads**

The **maxInstalls**  is introduced 7 days ago. Is it a bug that the stats do not match ?

**Comment by icarcal** [comment]
As @facundoolano said the initial purpose of the maxInstalls is different from what is reflecting now.

`minInstalls` reflects the `1,000,000,000+` installs at google play as you can see in the picture above:
<img width="147" alt="Screen Shot 2020-06-23 at 10 51 53" src="https://user-images.githubusercontent.com/11530480/85412080-94120f00-b53f-11ea-8a56-98206b2f171f.png">

`maxinstalls` field was removed, but the last time that I needed to change the mappings, I noticed another number that we could use at the `ds:6` field, that google play itself, allow us to read, for example:
```javascript
["1,000,000,000+",1000000000,2951198504,"1B+"]
```

The number 291198504 is not the total number of installs, it isn't accurate, and does not reflect the number of installs at google console.
Before adding this field I made the same validation, but figured out that is in fact a better number than 1000000000 itself, and close to what google console show us.
So I added it anyway.

You have two options:

- Ignore this number entirely and stick with `minInstalls`
- use `maxInstalls` as an approximately number of installs for whatever purpose you want

**Comment by usman10scorpio** [comment]
@facundoolano @icarcal is the histogram count equals to ratings ?

**Comment by icarcal** [comment]
@usman10scorpio 
The histogram represents this view from google play:
![Screen Shot 2020-06-26 at 14 33 55](https://user-images.githubusercontent.com/11530480/85885219-5d93f880-b7ba-11ea-990c-5819e873c1cf.png)

**Comment by AdityaAnand1** [comment]
I'm wondering what is everyone's use case, what are you guys using the install count for exactly?

**Comment by icarcal** [comment]
@usman10scorpio Can you provide your use case?

@AdityaAnand1 you can use this number to provide a more accuratte view of installs than let's say 1Mi+ or any round number.

**Comment by AdityaAnand1** [comment]
Yes.. I meant what's the use case of the install count (whatever the accuracy), as in, in what kind of products are you using these? Something like Appbrain or AppAnnie?

**Comment by facundoolano** [comment]
Issue automatically closed. Please create a new one if the issue persists
